### PR TITLE
LPS-181354 | Added Test automation FDSViews#CanCreateMultipleViews

### PR DESCRIPTION
**Pull request by andreluis2022**
Description:
Test done to create multiple views

Plan:
- And Add a Dataset
- And Adding a new view called "View Test" and the description field with "Test Description"
- And Adding a new view called "Bob Test" and the description field with "Bob Description"
- Then Assert that the 2 views are created

JIRA Ticket:
https://issues.liferay.com/browse/LPS-181354

### CODE REVIEW

**Comment by andreluis2022** [comment]
ci:test:frontend

File: portal-web/test/functional/com/liferay/portalweb/paths/portlet/frontenddataset/FrontendDataSetAdmin.path [hunk 1]
```diff
@@ -63,6 +63,11 @@
 	<td>//li[contains(@class,'list-group-item')]//div[contains(@class,'justify-content-center')]//p[(@class="list-group-title") and contains (.,'${entryName}')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>VIEW_ENTRY_NAME</td>
+	<td>//li[contains(@class,"list-group-item list-group-item-flex") and contains(.,'${key_name}')]</td>
+	<td></td>
+</tr>
```
Comment (mateusralv) [inline portal-web/test/functional/com/liferay/portalweb/paths/portlet/frontenddataset/FrontendDataSetAdmin.path:70]: I think `DATA_SET_VIEW_ENTRY_NAME` path does the same that `VIEW_ENTRY_NAME` if so, rename yours and delete the existing one.
```suggestion
<tr>
	<td>DATA_SET_VIEW_ENTRY_NAME</td>
	<td>//li[contains(@class,"list-group-item list-group-item-flex") and contains(.,'${key_name}')]</td>
	<td></td>
</tr>
```

**Review by mateusralv — APPROVED** [review]
Great! 🥳

**Comment by mateusralv** [comment]
Hi @andreluis2022, After the changes if the test passes the SF and the routine, we can forward

**Comment by andreluis2022** [comment]
ci:test:sf

**Comment by andreluis2022** [comment]
ci:forward

**Pull request closed without merging**